packages/core/src/client.ts

```diff
@@ -2171,7 +2171,7 @@ export class MedplumClient extends EventTarget {
       url.searchParams.set('_since', since);
     }
 
-    options.method = exportLevel ? 'GET' : 'POST';
+    options.method = 'GET';
```

Comment:
I think `GET` is the fetch default, so you might be able to delete this line entirely?